src/hres_eccb.py

```diff
@@ -0,0 +1,63 @@
+#!/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.01/bin/python
+
+"""
+Replace the land/surface fields in the ec_cb000 file with higher-resolution
+era5-land or BARRA2-R data (if requested).
+"""
+
+
+import argparse
+import pandas
+import shutil
+from pathlib import Path
+from datetime import datetime, timedelta
+
+import replace_landsurface_with_ERA5land_IC
+import replace_landsurface_with_BARRA2R_IC
+
+boolopt = {
+    "True": True,
+    "False": False,
+}
```

Comment:
What is this used for?

Reply:
This is not actually part of this PR. See my note on the branch coming from another feature branch, where the `src/hres_eccb.py` and `src/hres_ic.py` files were added ([see this PR](https://github.com/ACCESS-NRI/replace_landsurface/pull/17)).

I am not sure what they are for, as the `boolopt` variable doesn't seem to be used anywhere. 

I opened a [separate issue](#24) for this specific piece of the code.

Reply:
All good, thanks!